src/Routes/api.php

```diff
@@ -28,10 +28,8 @@
             Route::get('/{id}/people', HelpersRoute::getController('CourseController@people'))
                 ->middleware(BadasoAuthenticate::class)
                 ->name('people');
-        });
 
-        Route::group(['prefix' => 'courseuser', 'as' => 'courseuser.'], function () {
-            Route::get('/view', HelpersRoute::getController('CourseUserController@view'))
+            Route::get('/view', HelpersRoute::getController('CourseController@view'))
```

Comment:
usually, we use the path `/` only because it's pretty self-explanatory since the method is `GET`

Reply:
but the path `/` is already used by `add`, now that get courses is put under course

Reply:
notice that `add` is using `/` with POST method. We can still add a GET method for `/` to be used by `view`

Reply:
i have been enlightened orz
changing it rn

Reply:
done done